crates/bevy_core_pipeline/src/taa/taa.wgsl

```diff
@@ -1,17 +1,10 @@
 // References:
 // https://www.elopezr.com/temporal-aa-and-the-quest-for-the-holy-trail
+// https://alextardif.com/TAA.html
 // http://behindthepixels.io/assets/files/TemporalAA.pdf
 // http://leiy.cc/publications/TAA/TAA_EG2020_Talk.pdf
 // https://advances.realtimerendering.com/s2014/index.html#_HIGH-QUALITY_TEMPORAL_SUPERSAMPLING
 
-// Controls how much to blend between the current and past samples
-// Lower numbers = less of the current sample and more of the past sample = more smoothing
-// Values chosen empirically
-const DEFAULT_HISTORY_BLEND_RATE: f32 = 0.1; // Default blend rate to use when no confidence in history
-const MIN_HISTORY_BLEND_RATE: f32 = 0.015; // Minimum blend rate allowed, to ensure at least some of the current sample is used
-
-#import bevy_core_pipeline::fullscreen_vertex_shader
-
```

Comment:
This PR appears to include at least a partial version of the TAA from https://github.com/bevyengine/bevy/pull/8974. This should not be included unless https://github.com/bevyengine/bevy/pull/8974 is approved.

Reply:
❗ 

That really wasn't supposed to be the case.

Looks like I did a botched git commit that accidentally included those files:  fd58c77f82a0d985a26a512be7d3a60283808ff1

What likely happened is that I had those files checked out to try out the new TAA implementation to see how/if it affected transmission, and accidentally committed them. Let me revert that.

Reply:
Fixed!

Reply:
lol no worries :)